app/src/main/AndroidManifest.xml

```diff
@@ -39,25 +39,19 @@
                 <category android:name="android.intent.category.LAUNCHER" />
                 <category android:name="com.oculus.intent.category.2D" />
             </intent-filter>
-        </activity>
-        <activity
-            android:name="com.igalia.wolvic.VRBrowserActivity"
-            android:launchMode="singleInstance"
-            android:exported="true"
-            android:configChanges="density|keyboard|keyboardHidden|navigation|orientation|screenSize|uiMode|locale|layoutDirection"
-            android:windowSoftInputMode="stateAlwaysHidden">
```

Comment:
Are you sure the filter for configuration changes should be moved to the version check? Does VRBrowserActivity inherit them. Because if not we're going to have problems.